terraform/fargate-examples/sqs-dynamic-target-tracking/variables.tf

```diff
@@ -0,0 +1,85 @@
+variable "repository_owner" {
+  description = "The name of the owner of the Github repository"
+  type        = string
+  default     = "aws-ia"
```

Comment:
Erasing default value is recommended for customers to set their repo name easily

Reply:
done